client/landing/gutenboarding/variables.scss

```diff
@@ -0,0 +1,2 @@
+// Reusable style variables
+$gutenboarding-header-height: 64px;
```

Comment:
We currently use `$header-height` that is `56px` (which gets updated to `60px` when we upgrade Gutenberg?). 

p.s. took the liberty of introducing this file for reusable variables. Not sure if the right choice. I also prefixed with "gutenboarding" to avoid explicit rewrites. I was a little conflicted whether to use CSS variables instead e.g.

```
:some-root {
  --gutenboarding-header-height: 64px;
}
```

might just be the better choice?

Reply:
Curious what would be reasoning to use css variables for other than colors?

Reply:
This is confusing…

- Gutenberg uses the variable to build its styles: https://github.com/WordPress/gutenberg/search?l=SCSS&q=header-height
- We use the compiled results, we don't compile from sass source, right?
- Using the same variable doesn't help.

> I was a little conflicted whether to use CSS variables instead

The biggest difference seems to be is compile time vs. run time. SASS variables will be replaced at compile time, meaning they're mostly useful for sharing values and relating things we know at compile time. This seems to be one of those situations and SASS variable is a good choice.

For things like theming, CSS custom properties are great because we can change them on the fly at runtime and the CSS reacts. I don't think we need that here.

Reply:
One thing that is concerning is all the Gutenberg components that have the Gutenberg variable already baked in… they don't care that we change this value 😬

Reply:
> We use the compiled results, we don't compile from sass source, right?

You got me a little confused too. We import a SASS file (`assets/stylesheets/gutenberg-base-styles`), which results in importing the variables from `~@wordpress/base-styles/variables` that we end up using in our pre-processed styles. I am probably missing something or got lost in translation (happens all the time).

> One thing that is concerning is all the Gutenberg components that have the Gutenberg variable already baked in… they don't care that we change this value

Not sure I can translate this into an actual problem either. Why is this concerning? I thought my intention here would be clear (just find it easier to have own variable, rather than override something that's coming/available from outside) - but ok if this becomes a concern.

> The biggest difference seems to be is compile time vs. run time. SASS variables will be replaced at compile time, meaning they're mostly useful for sharing values and relating things we know at compile time. This seems to be one of those situations and SASS variable is a good choice.

^ this, pretty much @simison. You can ultimate have a layout built with variables that you can change at runtime. You'll probably get less value from them if used sporadically, but potentially a lot of value if thought out properly. Looked up and found https://googlechrome.github.io/samples/css-custom-properties/index.html

Reply:
> Not sure I can translate this into an actual problem either. Why is this concerning?

It's concerning because styles are being compiled with a value that is incorrect and different from the value we're using and expect.

A quick review of the [search results](https://github.com/WordPress/gutenberg/search?l=SCSS&q=header-height) suggests that `Modal` is the only component that needs to concern us. The rest are in edit-post which we're not relying on as far as I recall.

When we use a modal it will be slightly mispositioned, a bit too high, like we were seeing with content moving up under the header.

Reply:
I think that's a good point but all the more expected. We will adjust to match our designs when used (as we are doing).